src/opera/pge/dswx_ni/dswx_ni_pge.py

```diff
@@ -60,6 +61,32 @@ class DSWxNIPostProcessorMixin(DSWxS1PostProcessorMixin):
     _post_mixin_name = "DSWxNIPostProcessorMixin"
     _cached_core_filename = None
 
+    def _validate_output_product_filenames(self):
+        """
+        This method validates output product file names assigned by the SAS
+        via a regular expression.  The output product file names should follow
+        this conventions:
+
+         <PROJECT>_<LEVEL>_<PRODUCT TYPE>_<SOURCE>_<TILE ID>_<ACQUISITION TIMESTAMP>_
+         <CREATION TIMESTAMP>_<SENSOR>_<SPACING>_<PRODUCT VERSION>_<BAND INDEX>_
+         <BAND NAME>_<FILE EXTENSION>
```

Comment:
Should be `<BAND NAME>.<FILE EXTENSION>`

Reply:
good catch. Fixed